File: scala/scala-impl/testdata/refactoring/changeSignature/fromJava/NamedAndDefaultArgs_after.java
public class NamedAndDefaultArgs {
    public void foo(String s, boolean b, boolean b2) {}

    static class Inner extends NamedAndDefaultArgs {
        @Override
        public void foo(String s, boolean b, boolean b2) {
            System.out.println();
        }

        public void boo() {
            super.foo("3", false, true)
        }
    }
}

File: scala/scala-impl/testdata/refactoring/changeSignature/fromJava/NamedAndDefaultArgs_after.scala
class NamedAndDefaultArgsTest extends NamedAndDefaultArgs {
  override def foo(s: String = "aaa", b: Boolean, b2: Boolean): Unit = super.foo(s, b, true)

  foo(b = true, b2 = true)
  this foo(s = "bbb", b = true, true)
  this foo(b = true, b2 = true)
  foo(b = true, b2 = true)

  super.foo("", true, true)
}